### PR TITLE
Fix documentation typos

### PR DESCRIPTION
**Pull request by chocolateboy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,16 @@
 Beating C with 100 Lines of V.
```
Comment (waldyrious) [inline README.md]:
```suggestion
Beating C with 100 Lines of [V](https://vlang.io).
```

File: README.md [hunk 2]
```diff
@@ -274,11 +274,11 @@ Comparing the different implementations on the same files as before yields aston
...
+V itself is transpiled to C for optimized builds. You could almost say I compared C to C, but with one of the C implementations being almost like Go.
 
-Overall it was very interesting to see the comparison not only between C and V, but also GO and V. Using the same algorithms and functions obviously resulted in very similar results, but smaller differences could still be seen. Like in the other articles, this is not meant to be a V is better than C article. I wrote a very simplified version of the complete GNU wc. The only goal was to be faster, but still have the same results as GNU wc on pure ascii text.
+Overall it was very interesting to see the comparison, not only between C and V, but also Go and V. Using the same algorithms and functions obviously resulted in very similar results, but smaller differences could still be seen. Like in the other articles, this is not meant to be a V is better than C article. I wrote a very simplified version of the complete GNU wc. The only goal was to be faster, but still have the same results as GNU wc on pure ASCII text.
```
Comment (waldyrious) [inline README.md]:
```suggestion
Overall it was very interesting to see the comparison, not only between C and V, but also Go and V. Using the same algorithms and functions obviously resulted in very similar results, but smaller differences could still be seen. Like in the other articles, this is not meant to be a "V is better than C" article. I wrote a very simplified version of the complete GNU wc. The only goal was to be faster, but still have the same results as GNU wc on pure ASCII text.
```

**Review by waldyrious — APPROVED** [review]
Excellent work! I almost opened a PR to suggest precisely one of the typo fixes you're proposing, but I missed the bulk of your changes :)

I've added two inline comments with tweaks that I think might improve the text a bit more.

**Comment by schicho** [comment]
Oh boy, this is quite embarrassing...

Did proof read my on text once, but I guess when you are not a native speaker, you don't spot your own spelling errors and glance over simple typos.

Thank you so much for putting in that effort!


About the suggested changes: If you feel like adding those, I will wait a few more hours to merge this PR.

**Pull request merged**